services/web-app/components/mdx-page/mdx-page.js

```diff
@@ -117,7 +117,7 @@ const MdxPage = ({
           pictogram={pageHeader?.icon}
         />
       )}
-      {areTabsPresent && <PageTabs title={title} tabs={getTabData(tabs, baseSegment)} />}
+      {areTabsPresent && <PageTabs title="page tabs" tabs={getTabData(tabs, baseSegment)} />}
```

Comment:
Is this an HTML title attribute that shows up on operating system hover? If so, we typically use sentence case `Page tabs` per content guidelines:

https://next.carbondesignsystem.com/guidelines/content/writing-style#use-sentence-case-capitalization